#test exception
// division by 0
{
  return 1/0;
}
